pikaur/prompt.py

```diff
@@ -1,26 +1,76 @@
 import sys
+import tty
 from typing import Callable, List
 
+from pikaur.config import PikaurConfig
+
+from pikaur.args import PikaurArgs
+
 from .core import interactive_spawn
 from .i18n import _
 from .pprint import color_line, print_status_message
-from .args import PikaurArgs
 
 
-def ask_to_continue(text: str = None, default_yes=True, args: PikaurArgs = None) -> bool:
-    if text is None:
-        text = _("Do you want to proceed?")
+def get_answer(question, answers='Yn'):
```

Comment:
please add typing hint for the first argument

Reply:
Should `text` having a default in the parameters not cover that?

Reply:
`def get_answer(question, answers='Yn'):`

Reply:
Ooh sorry I see what you mean now. Do you want all parameters hinted or just ones without defaults?

Reply:
also return values not hinted as well